packages/client/src/components/LoginForm/ui/LoginForm.module.scss

```
@@ -0,0 +1,12 @@
+.login-form {
+  .error {
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+
+  .button-link-register {
+    text-align: center;
+    margin-top: 10px;
+    color: red;
```

```suggestion
    color: red;
```

Параметр не на что не влияет в текущей верстке

---

ок. спасибо!